internal/attestations/github.go

```diff
@@ -72,5 +72,5 @@ func (a *Attestations) SetGitHubPullRequestAuthorization(repo *git.Repository, e
 // GitHubPullRequestAttestationPath constructs the expected path on-disk for the
 // GitHub pull request attestation.
 func GitHubPullRequestAttestationPath(refName, commitID string) string {
-	return path.Join(refName, commitID)
+	return filepath.Join(refName, commitID)
```

Comment:
This path is used within Git to set up subtrees and eventually a blob. So, path.Join is fine.